docs/flutter/v2/features/background-handling.mdx

```diff
@@ -148,63 +148,94 @@ Add service inside `<application>` tag as follows:
         ...
     </activity>
 
-      // Set Service Type to capture Audio or Video as per your use-case. In most cases, only `microphone` access would be required. Then do not pass the `camera`
+      // Set Service Type to capture Audio or Video as per your use-case. In most cases, only `microphone` access would be required. Then do not pass the `camera`. `stopWithTask` will stop the notification if app is force killed.
```

Comment:
Mention `stopWithTask` outside the code snippet in the "Adding Service in" section as well